SlevomatCodingStandard/Sniffs/Classes/UnusedPrivateElementsSniff.php

```diff
@@ -622,6 +623,16 @@ private function findVisibilityModifierTokenPointer(File $phpcsFile, array $toke
 			return $visibilityModifiedTokenPointer;
 		}
 
+		if (in_array($visibilityModifiedToken['code'], [T_SELF, T_STRING], true)) {
+			$mightBeNullableTokenPointer = TokenHelper::findPreviousEffective($phpcsFile, $visibilityModifiedTokenPointer - 1);
+			$mightBeNullableToken = $tokens[$mightBeNullableTokenPointer];
```

Comment:
Please remove useless variable `$mightBeNullableToken`

Reply:
It's used in the ligne above, do you want me to inline it ?